tests/main/layout/task.yaml

```diff
@@ -7,9 +7,6 @@ details: |
     hooks get permissions to access those areas.
 
 prepare: |
-    echo "Ensure feature flag is enabled"
-    snap set core experimental.layouts=true
```

Comment:
Can we extend this test after exisitng checks to remove the snap, set `experimental.layouts=false` and check that it fails to install when the feature is missing?
And a little bit of a generic question (and not strictly related to this change): what happens when you disable the feature while having a snap that depends on it in the system?

Reply:
The same as if you have a parallel-instance snap installed, something undesired. We only check during a refresh.